django/cantusdb_project/users/admin.py

```diff
@@ -47,16 +55,19 @@ class UserAdmin(BaseUserAdmin):
                     "institution",
                     ("city", "country"),
                     "website",
-                )
+                ),
+                "description": "We have full name, first name, and last name defined in our User model which can be seen "
+                "in the admin area, but we are only using full name to represent users.",
```

Comment:
"...represent users _on the main site_"

Reply:
Perhaps something along the lines of: "you can enter a user's first and last name, but these are only ever displayed in the Admin area - on the main site, users' full names are always used rather than their first and last name"?